test/utilities/cliCommands.ts

```diff
@@ -225,3 +225,28 @@ export function removeEscapedCharacters(result: string): string {
 
   return resultJson;
 }
+
+
+export async function generateSfProject(
+  name: string,
+  path?: string | undefined,
+  template?: string | undefined,
+): Promise<SfCommandRunResults> {
+  const sfProjectGenerateResult = await runCliCommand(
+    'project:generate',
+    '--name',
+    name,
+    '--template',
+    template ?? 'standard',
+    '-d',
+    path ?? './'
```

Comment:
`./` would be process.cwd, which I believe is automation-tests root folder.

Also `./` will give windows trouble.